.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2
+jobs:
+  php72:
+    docker:
+      - image: circleci/php:7.2-cli-browsers-legacy
+    steps:
+      - checkout
+      - run: php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+      - run: php -r "if (hash_file('SHA384', 'composer-setup.php') === trim(file_get_contents('https://composer.github.io/installer.sig'))) { echo 'Installer verified'; } else { echo 'Installer invalid'; unlink('composer-setup.php'); } echo PHP_EOL;"
+      - run: php composer-setup.php
+      - run: php -r "unlink('composer-setup.php');"
+      - run: php ./composer.phar install
+      - run: php ./vendor/bin/phpunit tests
+
+  php71:
+    docker:
+      - image: circleci/php:7.1-cli-jessie-browsers-legacy
+    steps:
+      - checkout
+      - run: php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+      - run: php -r "if (hash_file('SHA384', 'composer-setup.php') === trim(file_get_contents('https://composer.github.io/installer.sig'))) { echo 'Installer verified'; } else { echo 'Installer invalid'; unlink('composer-setup.php'); } echo PHP_EOL;"
+      - run: php composer-setup.php
+      - run: php -r "unlink('composer-setup.php');"
+      - run: php ./composer.phar install
+      - run: php ./vendor/bin/phpunit tests
+
+  php70:
+    docker:
+      - image: circleci/php:7.0-cli-jessie-browsers-legacy
+    steps:
+      - checkout
+      - run: php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+      - run: php -r "if (hash_file('SHA384', 'composer-setup.php') === trim(file_get_contents('https://composer.github.io/installer.sig'))) { echo 'Installer verified'; } else { echo 'Installer invalid'; unlink('composer-setup.php'); } echo PHP_EOL;"
+      - run: php composer-setup.php
+      - run: php -r "unlink('composer-setup.php');"
+      - run: php ./composer.phar install
+      - run: php ./vendor/bin/phpunit tests
+
+  php56:
+    docker:
+      - image: circleci/php:5.6.38-cli-jessie-node-browsers
+    steps:
```

Comment:
I don't know much about it, but is it possible to move these `steps` into something like a macro/function?

Reply:
@jonrandy uhmmm I cannot find that kind of feature on their document.

Reply:
Google is your friend... https://discuss.circleci.com/t/circle-yml-macros-includes/13721/4

Reply:
lemmer try

Reply:
It worked?